packages/fxa-settings/README.md

```diff
@@ -2,16 +2,51 @@
+[Working with SVGs](#working-with-svgs)\
+[Metrics](#metrics)\
+— [Payload Data](#payload-data)\
+— [Event Logging](#event-logging)\
+[Testing and Mocks for Tests/Storybook](#testing-and-mocks-for-testsstorybook)\
+— [`AlertBar` or `AlertExternal`](#components-that-use-alertbar-or-alertexternal)\
+— [`useAccount`](#components-that-use-useaccount)\
+[Storybook](#storybook)\
+[License](#license)
+
+## Relevant ADRs
+
+- [Create a New React Application for the Settings Redesign Project](https://github.com/mozilla/fxa/blob/main/docs/adr/0011-create-new-react-app-for-settings-redesign.md)
+- [React Toolchain for Settings Redesign](https://github.com/mozilla/fxa/blob/main/docs/adr/0013-react-toolchain-for-settings-redesign.md)
+- [Use GraphQL and Apollo for Settings Redesign](https://github.com/mozilla/fxa/blob/main/docs/adr/0016-use-graphql-and-apollo-for-settings-redesign.md)
+- [Switch from OAuth2 to Sharing sessionToken with GraphQL Server for Settings Redesign Auth](https://github.com/mozilla/fxa/blob/main/docs/adr/0017-switch-settings-auth-to-sessiontoken.md)
+- [Use Utility-First CSS (Tailwind) with Custom SCSS](https://github.com/mozilla/fxa/blob/main/docs/adr/0018-use-tailwind-with-custom-scss.md)
+- [Use Existing InternJS for Functional Testing in Settings V2](https://github.com/mozilla/fxa/blob/main/docs/adr/0021-use-internjs-testing.md)
+
 ## Development
 
 - `yarn start|stop|restart` to start, stop, and restart the server as a PM2 process
 - `yarn build` to create a production build
 - `yarn test` to run unit tests
 - `yarn storybook` to open Storybook
 
-### GQL and REST API Calls
+## GQL and REST API Calls
 
-FxA Settings communicates primarily with the FxA GraphQL API through use of [Apollo Client](https://www.apollographql.com/docs/react/) to indirectly interact with the `fxa-auth-server`. Requests that simply retrieve data are called queries and all other interaction requests are called mutations. See the [GQL documentation](https://graphql.org/learn/) to learn more. See ([he documentation for `fxa-graphql-api`](https://github.com/mozilla/fxa/tree/main/packages/fxa-graphql-api) to connect to the playground, a place to view the API docs and schema, and to write and test queries and mutations before using them in a component.
+FxA Settings communicates primarily with the FxA GraphQL API through use of [Apollo Client](https://www.apollographql.com/docs/react/) to indirectly interact with the `fxa-auth-server`. Requests that simply retrieve data are called queries and all other interaction requests are called mutations. See the [GQL documentation](https://graphql.org/learn/) to learn more. See [the documentation for `fxa-graphql-api`](https://github.com/mozilla/fxa/tree/main/packages/fxa-graphql-api) to connect to the playground, a place to view the API docs and schema, and to write and test queries and mutations before using them in a component.
```

Comment:
This change was  just a typo fix from "to learn more. See ([he" to "to learn more. See [the"